packages/react-native/android/build.gradle

```diff
@@ -87,3 +87,10 @@ ReactNative.shared.applyPackageVersion()
 ReactNative.shared.applyDefaultExcludes()
 ReactNative.module.applyAndroidVersions()
 ReactNative.module.applyReactNativeDependency("api")
+rootProject.allprojects {
```

Comment:
@christocracy - you might like this, removes an error-prone install step for react-native repos with local maven repositories to load aars, appears to work fine for us here, could clean up my app build files from background-fetch and geolocation-android :-)

Reply:
Whoa, cool!!